cmd/os-reliable.go

```diff
@@ -95,6 +95,8 @@ func renameAll(srcFilePath, dstFilePath string) (err error) {
 			return fmt.Errorf("%s (%s)->(%s)", errCrossDeviceLink, srcFilePath, dstFilePath)
 		case os.IsNotExist(err):
 			return errFileNotFound
+		default:
+			return err
 		}
 	}
 	return err
```

Comment:
It will look cleaner, if it is changed to `return nil`

Reply:
@cristaloleg,
Could you please incorporate @kannappanr's comment?
Thanks